lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -1054,3 +1054,58 @@ yum:
   remove:
     config_set: "install deactivate %s"
 
+vxlan_global:
+  feature:
+    config_get: "show running | i ^feature"
+    config_get_token: '/^feature fabric forwarding$/'
+    config_set: "<state> feature fabric forwarding"
+
+  dup_host_ip_addr_detection:
+    config_get: 'show running fabric forwarding all'
+    config_get_token: '/^fabric forwarding dup-host-ip-addr-detection (\d+) (\d+)$/'
+    config_set: "<state> fabric forwarding dup-host-ip-addr-detection <host_moves> <timeout>"
+
+  dup_host_ip_addr_detection_host_moves:
+    default_value: 5
+
+  dup_host_ip_addr_detection_timeout:
+    default_value: 180
+
+  anycast_gateway_mac:
+   config_get: 'show running fabric forwarding all'
+   config_get_token: '/^fabric forwarding anycast-gateway-mac (\S+)$/'
+   config_set: "%s fabric forwarding anycast-gateway-mac %s"
+
+  dup_host_mac_detection:
+    config_get: 'show running | i l2rib'
+    config_get_token: '/^l2rib dup-host-mac-detection (\d+) (\d+)$/'
+    config_set: "l2rib dup-host-mac-detection <host_moves> <timeout>"
+
+  dup_host_mac_detection_default:
+    config_set: "l2rib dup-host-mac-detection default"
+
+  dup_host_mac_detection_host_moves:
+    default_value: 5
+
+  dup_host_mac_detection_timeout:
+    default_value: 180
+
+
```

Comment:
no - you added ~20 blank lines to the end of the file (lines 1093 - 1111). You need to remove those.